core/tutorialbar.py

```diff
@@ -53,13 +56,15 @@ def get_course_links(self, url: str) -> List:
         :return: list of pages on tutorialbar.com that contain Udemy coupons
         """
         response = requests.get(url=url)
+
         soup = BeautifulSoup(response.content, "html.parser")
+
         links = soup.find("div", class_="rh-post-wrapper").find_all("a")
-        self.last_page = links[-2].text
```

Comment:
We need this line for the script to work properly:
`self.last_page = links[-2].text`

Reply:
:confused: What do you mean? I think it is working properly...

Reply:
@cullzie ?

Reply:
Ye it needs to be added back in.

The print is wrong because we never set the last_page variable now.
So it shows `Page: 1 of None scraped` and so on:
```
Please Wait: Getting the course list from tutorialbar.com...
Page: 2 of None scraped

```

Reply:
Oh ok thanks!